src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Added internal `InstrumentationScope`, `LoggerProvider`, `Logger`,
+  `LoggerProviderBuilder`, `LogRecordSeverity`, and
+  `IDeferredLoggerProviderBuilder` types and moved `LogRecordData` and
+  `LogRecordAttributeList` types from SDK.
+  ([#4422](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4422))
```

Comment:
Do we need a changelog yet? Maybe we do a changelog when we've exposed stuff?

Reply:
Since this has no visible user-effect, its okay to not have it. OR keep it, but explicitly call out the changes are internal and has no impact on end users.

Reply:
I think "it's not okay to have it since the user won't notice any change".

Reply:
I removed the CHANGELOG entry